frontend/src/js/ui/components/generator-config.jsx

```diff
@@ -19,7 +19,7 @@ export default () => ({
 				</div>
 				<div className='divider' />
 				{vnode.attrs.config.map((val) => {
-					if (!Types[val.type] || !vnode.attrs.value[val.key]) {
+					if (!Types[val.type]) {
```

Comment:
If there was a specific reason for adding this, we should probably check `undefined` or another value rather than the falsiness of the value